source/includes/administration/_resource_commitments.md

```diff
@@ -0,0 +1,343 @@
+    "startDate": "2019-05-16T00:00:00Z",
+    "endDate": "2019-05-24T00:00:00Z",
+    "pricingMethod": "FIXED_MONTHLY",
+    "currency": "CAD",
+    "monthlyPrice": 20.0,
+    "effectiveDiscount": 92.39,
+    "billableHoursPerMonth": 730,
+    "billingDay": 16,
+    "resourceCommitmentDetails": [
+      {
+        "id": "08125b3c-3b25-4585-910b-c1a0dc53010a",
+        "name": "CustomDoF",
+        "primaryType": "6",
+        "secondaryType": "CustomDoF",
+        "commitment": 0,
+        "discountPercent": 0,
+        "unit": "gigabyte",
+        "labelKey": "cloudstack.volume"
+      },
+      {
+        "id": "0bf1a48d-f807-4821-8247-2eb9a3128091",
+        "primaryType": "3",
+        "commitment": 0,
+        "discountPercent": 0,
+        "unit": "unit",
+        "labelKey": "cloudstack.ipAddress"
+      },
+      { ... },
+      { ... },
+      { ... }
+    ],
+    "serviceConnection": {
+      "name": "dev1-cloudstack",
+      "id": "cb511a38-eab5-4a2e-969b-04e129c98415",
+      "type": "cloudstack"
+    },
+    "organization": {
+      "name": "System",
+      "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e"
+    },
+    "version": 1,
+    "deleted": false
+  }, 
+  {
+    "id": "51c01cfe-f0d5-4a3d-841d-35ec2b7df9cc",
+    "displayName": "CRB-SYSTEM-2",
+    ....
+  }]
+}
+```
+List all the resource commitments that you have access to. You can only view resource commitments if you have been assingned either the `View usage` or the `Manage commitments` permission. If you have `View usage` access then you will only be able to see the commitments specific to your organization. Having the `Manage commitments` permission lets you see all resource commitments in the system.
+
+An array of resource commitments with the following attributes are returned.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the resource commitment
+`displayName`<br/>*string* | The name of the resource commitment
+`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the resource commitment was created
+`startDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) from when the resource commitment becomes active
+`endDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) until when the resource commitment will be active<br/>*optional*: `if not present then the commitment has no end date`
+`pricingMethod`<br/>*string* | One of the two following options that define how the effective price for the resource commitment is calculated<br/>*FIXED_MONTHLY*: `a fixed price per billing cycle`<br/>*VARIABLE_UTILITY_DISCOUNT*: `pricing with custom discount on specific resources`
+`currency`<br/>*string* | The currency type based on which the pricing is defined
+`monthlyPrice`<br/>*number* | The effective monthly price for this resource commitment
+`effectiveDiscount`<br/>*number* | The effective discount on this resource commitment
```

Comment:
It should mention that this is a percentage

Reply:
Fixed in 738496526a7aa1f45bc86f95f7c32cc0e4ed3320